ngraph/frontend/onnx_import/src/op/upsample.cpp

```diff
@@ -152,47 +151,17 @@ namespace ngraph
                     const auto data = inputs.at(0);
                     const auto scales = inputs.at(1);
 
-                    const auto data_shape = data.get_partial_shape();
-                    const auto scales_shape = scales.get_partial_shape();
-
                     const auto mode = node.get_attribute_value<std::string>("mode", "nearest");
                     check_mode_support(node, mode, version_9);
 
+                    const auto data_shape = data.get_partial_shape();
+                    const auto scales_shape = scales.get_partial_shape();
```

Comment:
you can consider changing that to const reference

Reply:
considered and applied ;)